pkg/services/blockfetcher/blockfetcher.go

```diff
@@ -59,12 +62,12 @@ type Service struct {
 	log               *zap.Logger
 	cfg               config.NeoFSBlockFetcher
 	stateRootInHeader bool
-
-	chain         Ledger
-	pool          poolWrapper
-	enqueueBlock  func(*block.Block) error
-	enqueueHeader func(*block.Header) error
-	account       *wallet.Account
+	headerSize        int
```

Comment:
Comment, and group it with `stateRootInHeader` since it's more like a chain setting.